js/data/scripts/createTweetsOl.js

```diff
@@ -0,0 +1,12 @@
+"use strict";
+
+function createTweetOl(tweet) {
+	var ol = document.createElement('ol');
+	var el = document.querySelector('div');
+	el.appendChild(ol);
```

Comment:
oh wait no, they're not useless, but they're not what was asked for.
This function should return an `<ol>` and the calling context should be the one deciding where the ol should be appended.